lib/msal-core/package.json

```diff
@@ -38,8 +38,9 @@
     "@types/chai": "^4.1.7",
     "@types/handlebars": "4.0.33",
     "@types/mocha": "^5.2.6",
-    "@types/node": "^11.11.5",
+    "@types/node": "^11.13.4",
```

Comment:
Any breaking changes?

Reply:
no, its a minor update, just wanted to update types for node libs